### DISCUSSION TITLE
怎么调整连接桩的top 和 bottom的平均分配的间距

### DISCUSSION THREAD

**Discussion by ljf0910** [antvis/X6#2650] [Q&A]
![image](https://user-images.githubusercontent.com/53027879/189049697-4dc3a361-52c7-427a-97ed-c6e60169d08d.png)
想把点上面的两个点距离调小点，又不影响单个点在中间
![image](https://user-images.githubusercontent.com/53027879/189049832-04c2e00c-959c-47f5-9754-f74231752b46.png)

**Comment by zmcode** [comment]
https://x6.antv.vision/zh/docs/api/registry/port-layout#registry
通过数量自行算, 链接桩最终是通过circle来渲染的 还可以配置offset 这个attr来配置
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute#core
